rune/libenclave/attestation/sgx/ias/ias.go

```diff
@@ -335,7 +345,7 @@ func checkVerificationReport(resp *http.Response, quote []byte, nonce string) (*
 	status.timestamp = report.Timestamp
 	status.quoteStatus = report.IsvEnclaveQuoteStatus
 
-	if report.Version != apiVersion {
+	if report.Version != (uint32)(apiVersion) {
```

Comment:
So why not define apiVersion as u32?

Reply:
> So why not define apiVersion as u32?

Because the function strconv.ParseUint(APIVersion, 10, 32) returns (uint64, error).

Reply:
> > So why not define apiVersion as u32?
> 
> Because the function strconv.ParseUint(APIVersion, 10, 32) returns (uint64, error).

OK.